Print NA states

mkiang opened the pull request:
Addresses https://github.com/hrbrmstr/statebins/issues/14 but I haven't tested it very thoroughly and requires the user to specify an NA label, but it might be preferable to do it another way...

hrbrmstr commented:
#ty! I'll take a look at it (likely in the early AM tomorrow). Could you also update the DESCRIPTION file and add yourself as a contributor?

mkiang commented:
Sure. I updated to try to match your style guide and added myself to the `DESCRIPTION` but given the minor nature of the edit, I'm happy with an uncredited PR. Thanks for the package — hoping to use it on a publicly-available conference presentation this month.

mkiang commented:
Realized I didn't modify the `geom`. Did so and here is some sample code to make sure it works:

```
library(statebins)
library(dplyr)
library(ggplot2)
data(USArrests)

USArrests$state <- rownames(USArrests)
USArrests$Assault[1:4] <- NA

statebins(USArrests, value_col="Assault", name = "Assault", round=TRUE) +
    theme_statebins(legend_position="right")

ggplot(USArrests, aes(state = state, fill = Assault)) + 
    geom_statebins() + 
    theme_statebins(legend_position="right") + 
    coord_equal()
```
